src/js/player.js

```diff
@@ -300,7 +300,17 @@ class Player extends Component {
     if (tag.parentNode) {
       tag.parentNode.insertBefore(el, tag);
     }
-    Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    //Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    // append the DOM el function to make use of our component.addChild method
+    // to properly be injected into the children Array
+    tag.el = function () {
```

Comment:
It's because `addChild` defaults the insertion index to the length of the children array but if we insert the element manually, it won't end up in the children array so items will end up getting inserted before it into the DOM.
The workaround would be to insert it manually into the dom and then insert it manually into the children's array:
```js
Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
this.children_.unshift(tag);
```